nltk/tag/stanford.py

```diff
@@ -204,6 +205,79 @@ def parse_output(self, text, sentences):
 
         raise NotImplementedError
 
+class CoreNLPTagger(CoreNLPParser, TaggerI):
+    def __init__(self, url='http://localhost:9000', encoding='utf8'):
```

Comment:
Actually, I want to shield the user from abusing the *args and **kwargs here so that nothing erroneous gets through and restrict the CoreNLPTagger to only the tagging task. 

I'm not sure which is more Pythonic though.